web_app/templates/home.html

```diff
@@ -0,0 +1,65 @@
+{% extends "bootstrap_5_layout.html" %}
+{% set active_page = "home" %}
+
+{% block content %}
+
+    <h1>Colonial Courses</h1>
+
+    <p class="lead">What courses are available?</p>
+
+    <div class="container" style="max-width: 350px;">
+        <div class="card card-body bg-light mt-3">
+            <form action="/search" method="POST">
+                <div class="mb-3">
+                    <label for="term-input" class="form-label">Select Term:</label>
+                    <select id="term-input" class="form-select" name="term_id">
+                        <option value="202401" selected>Spring 2024</option>
```

Comment:
todo: mechanism for updating for new semesters (we don't want to bump the semester each time. either dynamically generate, or instruct users to pass the semester code instead "202403".